index.html

```diff
@@ -25,7 +25,7 @@
             <div id="close" class="fas fa-times"></div>
 
             <a href="#" class="nav_item">Beranda</a>
-            <a href="#" class="nav_item">about</a>
+            <a href="#" class="nav_item">tentang</a>

```

Comment:
ini tur ubah yang ini jadi kentang
```suggestion
            <a href="#" class="nav_item">kentang</a>

```